src/App.js

```diff
@@ -1,17 +1,50 @@
-import { useState } from 'react';
-import './App.css';
-import Calculator from './component/Calculator';
-import calculate from './logic/calculate';
+/* eslint-disable  */
```

Comment:
- [ ] It would be nice not to disable linting on the file. It beats the primary purpose as to why they were being introduced into the project in the first place. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. If part of your code breaks the standard rules, this can pose a problem.

NOTE: 
It is not acceptable to disable linting entirely. If you've got one problem that's hard to deal with, you are allowed to disable only the single rule that's in line with that problem and leave a notice of it in the PR explaining why you disabled that particular rule.

I have enabled linting in all your JS files and found out all the errors are simple ones that majorly have to do with code styling. These can be easily taken care of. Always remember to run `npx eslint . --fix` after saving your files to help you solve more than 90% of code-styling-related problems.

Please reenable linting for your JS files and correct the errors. Files in question:
* App.js
* Navbar.js
* Calculator.js
* index.js
* ErrorPage.js
* Home.js
* Quotes.js